onnxruntime/contrib_ops/cpu/transformers/logits_processor.cc

```diff
@@ -9,6 +9,8 @@ namespace onnxruntime {
 namespace contrib {
 namespace transformers {
 
+static int beam_search_iteration;
```

Comment:
The iteration can be stored in ISequences (For example, let Sequences stored the start sequence length, then it can be deduced from current sequence length - start sequence length). In this way, there is no need to store it here.

Reply:
As discussed, added a comment here as it is more intuitive and readable to have a common var.